src/common/codeUtil.ts

```diff
@@ -48,4 +51,85 @@ export class CodeUtil {
 
         return regs;
     }
+
+    /** Align variable assignment by = operator in selected text
+     * @param selection Text selection in editor
+     */
+    public static alignTextAssignOperator(selection: Selection): string {
+        const document = vscode.window.activeTextEditor.document;
+        let maxPosition = 0; // Right-most = operator position in line from all assignments
+        for (
+            let lineIndex = selection.start.line;
+            lineIndex <= selection.end.line;
+            lineIndex++
+        ) {
+            const line = this.normalizeLineAssignOperator(
+                document.lineAt(lineIndex).text,
+            );
+
+            // Find right-most = operator position
+            let position = line.search('='); // = operator position
+            if (position > maxPosition) {
+                maxPosition = position;
+            }
+        }
+
+        let text = '';
+        for (
+            let lineIndex = selection.start.line;
+            lineIndex <= selection.end.line;
+            lineIndex++
+        ) {
+            let line = document.lineAt(lineIndex).text;
+            text += this.alignLineAssignOperator(line, maxPosition);
+            if (lineIndex !== selection.end.line) {
+                text += '\n';
+            }
+        }
+
+        return text;
+    }
+
+    /** Remove comment, extra spaces around = operator and
+     * add spaces around = and := operators (if they missing).
+     * Remove extra spaces, but not touch leading and trailing spaces.
+     * @param original Original line of code
+     */
+    public static normalizeLineAssignOperator(original: string): string {
```

Comment:
How does this work on the following lines of code? Want to make sure we're handling quoted `=` correctly:

- x := "1+1=2"
- val = "="
- withSpaces = "x = y"